components/automate-ui/src/app/pages/project/rules/project-rules.component.html

```diff
@@ -51,53 +51,57 @@
               <label>
                 <span class="label">{{ getAttributeLabel() }}</span>
                 <select ngDefaultControl formControlName="attribute">
-                  <option 
+                  <option
                     *ngFor="let option of attributes[ruleForm.get('type').value.toLowerCase()]; let i = index"
                     [value]="option.key"
                   >{{ option.value }}</option>
                 </select>
               </label>
+              <chef-error
+                *ngIf="ruleForm.get('conditions').controls[i].get('attribute').hasError('required') && ruleForm.get('conditions').controls[i].get('attribute').touched"
+              >Attribute is required</chef-error>
             </chef-form-field>
 
             <chef-form-field class="attribute-field">
               <label>
                 <span class="label">Operator</span>
-                <select ngDefaultControl
-                  formControlName="operator"
-                  (change)="updateConditionValues(condition)"
-                >
-                  <option value='EQUALS'>equals</option>
-                  <option value='MEMBER_OF'>member of</option>
+                <select ngDefaultControl formControlName="operator">
+                  <option *ngFor="let operator of operators"
+                    [value]="operator.key"
+                  >{{ operator.value }}</option>
```

Comment:
I prefer not to have duplicate source of truth, so moved these values to the backing code.